### ISSUE TITLE
Fix KubePersistentVolumeFullInFourDays alert

### ISSUE THREAD

**Issue by mrIncompetent** [kubermatic/kubermatic#2162]
**User Story**
We're receiving the `KubePersistentVolumeFullInFourDays` alert on a regular basis.
As the affected disks are not even close to being full, the alert should be fixed.

**Comment by kron4eg** [comment]
```
alert: KubePersistentVolumeFullInFourDays
expr: kubelet_volume_stats_available_bytes{job="kubelet"}
  and predict_linear(kubelet_volume_stats_available_bytes{job="kubelet"}[6h],
  4 * 24 * 3600) < 0
for: 5m
```
`for: 5m` is too low, should be changed to at least 1 hour or even more

**Comment by mrIncompetent** [comment]
sgtm

**Comment by kron4eg** [comment]
In upstream kubernetes-mixins it was addressed (although in different way).

**Comment by mrIncompetent** [comment]
Reopening this as we still hit a lot of those alerts on cloud

**Comment by kron4eg** [comment]
@mrIncompetent I just discovered that asia cluster doesn't have updated alerting rules

**Comment by kron4eg** [comment]
Because of #2204 asia cluster has heavy outdated monitoring configs, this issue will be fixed once #2204 is fixed.

**Comment by xrstf** [comment]
#2204 has been merged and deployed and alerts on cloud have stopped on October 12th.
